apps/hash-frontend/src/pages/shared/entities-table.tsx

```diff
@@ -414,9 +439,12 @@ export const EntitiesTable: FunctionComponent<{
       },
       {
         columnKey: "lastEditedBy",
-        filterItems: lastEditedByUsers.map(({ accountId, preferredName }) => ({
-          id: accountId,
-          label: preferredName ?? "Unknown User",
+        filterItems: lastEditedByActors.map((actor) => ({
+          id: actor.accountId,
+          label:
+            ("displayName" in actor
+              ? actor.displayName
+              : actor.preferredName) ?? "Unknown Actor",
```

Comment:
I know we've spoken about this before but I'm wondering again if it would be simpler just to make it `displayName` for both machines and human actors in the frontend to save all these ternaries. Not suggesting any changes now but interested in your view having done this PR and whether we do it next time we're touching this stuff

Reply:
I think it'd make sense and agree save a lot of unnecessary hassle, but I'd still keep `preferredName` on `MinimalUser` as it's the property on the entity even though they'll have the same value. We could probably do the same for `MinimalOrg` as well.